### ISSUE TITLE
Some links don't display "reader mode" pages properly

### ISSUE THREAD

**Issue by ekianjo** [go-shiori/shiori#171]
Most of the time it works just fine with Shiori, but there are some pages, like this one: https://codingclubuc3m.rbind.io/post/2019-09-24/, where clicking on it after bookmarking it results in a white page in Firefox and Chromium. The strange thing is that hitting F5 to refresh the page will show the page for a brief second until it refreshes to a white page. Any idea as to what may be happening?

**Comment by RadhiFadlillah** [comment]
Nice catch, thanks. It should be fixed by [b315c48](https://github.com/go-shiori/shiori/commit/b315c48acf857f43266bf724b21caa7702ebc87d).

By the way, this issue happened because in your link it uses double braces inside the article:

```
## Equations

This summation expression $\sum_{i=1}^n X_i$ appears inline.

$$f(y|N,p) = {{N}\choose{y}} \cdot p^y \cdot (1-p)^{N-y}$$
```

This conflict with default delimiter that used by Vue, which make it confused when rendering it. Fortunately Vue has [`v-pre`](https://vuejs.org/v2/api/#v-pre) directives which solve this issue.

**Comment by ekianjo** [comment]
Thanks for fixing this! I will close the issue :)
